wagtail/core/migrations/0001_squashed_0016_change_page_url_path_to_text_field.py

```diff
@@ -339,6 +330,10 @@ class Migration(migrations.Migration):
                 )
                 ),
             ],
+            options={
+                'swappable': swapper.swappable_setting('wagtailcore', 'Site'),
+            },
+
         ),
         migrations.AlterUniqueTogether(
```

Comment:
I hadn't noticed this before, but this `AlterUniqueTogether` item (and the `AlterModelOptions` one below)... would they apply to the default model only, or would they apply to a custom model also?